libs/island-ui/theme/src/lib/theme.ts

```diff
@@ -98,10 +97,13 @@ export const theme = {
       solid: 'solid',
     },
     radius: {
-      standard: '4px',
-      large: '8px',
-      xl: '24px',
-      circle: '50%',
+      standard: `${radius.xs.$value ?? 4}px`, // TODO: check with designers to keep as standard
+      large: `${radius.default.$value ?? 8}px`, // TODO: check with designers to keep as large
+      md: `${radius.md.$value ?? 12}px`,
+      lg: `${radius.lg.$value ?? 16}px`,
+      xl: '24px', // TODO: should be added to figma?
+      circle: '50%', // TODO: should be added to figma?
+      full: `${radius.full.$value ?? 9999}px`,
```

Comment:
`circle` and `full` achieve the same thing, should we omit either one?

Reply:
good spot! for sure, any preference on which to omit? perhaps circle as it has not been introduced in figma

Reply:
will be addressed in the followup PR

Reply:
> perhaps circle as it has not been introduced in figma

Sounds good!